Model/User.php

```diff
@@ -190,9 +176,7 @@ public function unserialize($serialized)
             $this->salt,
             $this->usernameCanonical,
             $this->username,
-            $this->expired,
             $this->locked,
-            $this->credentialsExpired,
```

Comment:
Isn't cache supposed to be cleared? Otherwise i think we can accept it because 2.0 is a major release.